source/common/common/fine_grain_logger.cc

```diff
@@ -174,9 +169,6 @@ spdlog::logger* FineGrainLogContext::createLogger(const std::string& key)
 void FineGrainLogContext::updateVerbosityDefaultLevel(level_enum level) {
   {
     absl::WriterMutexLock wl(&fine_grain_log_lock_);
-    if (level == verbosity_default_level_) {
```

Comment:
context for removing this in this change?

Reply:
This is a slight behavior change: setting the default level will clear the existing verbosity updates and update all the loggers. People may say why the update does not take effect if they keep posting the same update as the previous default if there is glob verbosity update. I am not sure which one is better for if we want to keep the glob verbosity update if we are changing the default level. What's your opinion?

Reply:
Looking at the component level logger it would do the same as this new behavior -- imo consistency here makes sense. Given that is the more mature system that folks would migrate from this new behavior makes sense to me.